js/picture-modal.js

```diff
@@ -0,0 +1,57 @@
+import { isEscapeKey } from './util.js';
+
+const body = document.querySelector('body');
+const fullScreenImageDisplay = document.querySelector('.big-picture');
+const exitFullScreenImage = fullScreenImageDisplay.querySelector('.big-picture__cancel');
+const imageComments = fullScreenImageDisplay.querySelector('.social__comment-count');
+const commentsDownloadButton = fullScreenImageDisplay.querySelector('.comments-loader');
+const fullScreenImage = fullScreenImageDisplay.querySelector('.big-picture__img');
+const likesCount = fullScreenImageDisplay.querySelector('.likes-count');
+const commentsCount = fullScreenImageDisplay.querySelector('.comments-count');
+const photoDescription = fullScreenImageDisplay.querySelector('.social__caption');
+const commentsList = fullScreenImageDisplay.querySelector('.social__comments');
+const commentItem = fullScreenImageDisplay.querySelector('.social__comment');
+
+const onDocumentKeyDown = (evt) => {
+  if (isEscapeKey(evt)) {
+    evt.preventDefault();
+    imageModalClose(evt);
+  }
+};
+
+const imageModalOpen = () => {
+  fullScreenImageDisplay.classList.remove('hidden');
+  imageComments.classList.add('hidden');
+  commentsDownloadButton.classList.add('hidden');
+  body.classList.add('modal-open');
+  exitFullScreenImage.addEventListener('click', imageModalClose);
+  document.addEventListener('keydown', onDocumentKeyDown);
+};
+
+const renderingImage = (obj) => {
+  fullScreenImage.querySelector('img').src = obj.url;
```

Comment:
название функции поправь, плюс параметр лучше назвать более семантично

```suggestion
const renderImage = (image) => {
  fullScreenImage.querySelector('img').src = image.url;
```